pkgs/by-name/ra/raycast/package.nix

```diff
@@ -40,14 +49,26 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   passthru.updateScript = lib.getExe (writeShellApplication {
     name = "raycast-update-script";
     runtimeInputs = [
+      cacert
       curl
       jq
-      common-updater-scripts
+      nix
     ];
     text = ''
       url=$(curl --silent "https://releases.raycast.com/releases/latest?build=universal")
       version=$(echo "$url" | jq -r '.version')
-      update-source-version raycast "$version" --file=./pkgs/by-name/ra/raycast/package.nix
+
+      arm_url="https://releases.raycast.com/releases/$version/download?build=arm"
+      x86_url="https://releases.raycast.com/releases/$version/download?build=x86_64"
+
+      arm_hash=$(nix hash to-sri --type sha256 "$(nix-prefetch-url "$arm_url")")
```

Comment:
This looks like [recursive-nix](https://nix.dev/manual/nix/2.24/development/experimental-features.html?highlight=recursive#xp-feature-recursive-nix) to me
IIRC experimental features should be avoided in nixpkgs if possible
Relevant discussion: https://discourse.nixos.org/t/best-approach-for-updatescript-when-there-are-multiple-hashes/42897

Reply:
As a workaround, I'm using `curl` and `openssl` to create the same output that those two nix commands would have given me